Backend/src/modules/articles/articles.service.ts

```
@@ -16,32 +17,154 @@ export class ArticlesService {
+        const match = block.src ? block.src.match(/^BLOCK_IMAGE_(\d+)$/) : null;
+        if (match && blockFiles) {
+          const index = parseInt(match[1], 10);
+          const file = blockFiles[index];
+          if (file) {
+            const newImageUrl = await this.imageService.uploadImage(file);
+
+            // Delete old image if existed
+            if (
+              oldBlocks &&
+              oldBlocks[i] &&
+              oldBlocks[i].src &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)